### ISSUE TITLE
OP1093 Add prefix to Subject line when processing contact form

### ISSUE THREAD

**Issue by nicktrosnant** [u3a-siteworks-development/u3a-siteworks-contact-form#20]
Raised in [this post in the forum](https://u3awpdev.org.uk/viewtopic.php?t=626).

A WM reported contact message recipient deleting messages sent via the form suspecting them to be fraudulent because they had very minimal subject lines such as "venue" or "Question".

Discussion concludes that adding a standard prefix such as "u3a enquiry: " to the subject line would be helpful.

**Comment by mike99christie** [comment]
I agree with Andy Henderson that it would be better for the nervous recipient to be told to check who the email is from. If it from their u3a's website than it is safe to open it and read what it says. Worrying about a short meesage subject won't help, and adding "mytown u3a enquiry" is something that they could imagine was a spoof, and will make a genuine subject message even longer.

**Comment by nicktrosnant** [comment]
A well reasoned [suggestion from the forum](https://u3awpdev.org.uk/viewtopic.php?p=2539#p2539).

"The challenge is how to get more meaningful subject lines. Here are two suggestions:

- Introduce a sensible minimum length for subject lines. (Say 16 characters).
- Pre-fill the subject line with a suitable default - such as <u3a name> u3a Query or even <u3a name> u3a <Group Name> Query

Either would be a sufficient subject even if the user didn't refine it further."
